genai-perf/tests/test_csv_exporter.py

```diff
@@ -335,3 +341,151 @@ def test_invalid_goodput_csv_output(
                 f"Expected file {expected_filename} not found in written data."
             )
         assert returned_data[-1] == expected_content
+
+    def test_triton_telemetry_metrics_output_success(
+        self, monkeypatch, mock_read_write: pytest.MonkeyPatch
+    ) -> None:
+        argv = [
+            "genai-perf",
+            "profile",
+            "-m",
+            "model_name",
+            "--service-kind",
+            "triton",
+            "--streaming",
+            "--server-metrics-url",
+            "http://tritonserver:8002/metrics",
+        ]
+        monkeypatch.setattr("sys.argv", argv)
+        args, _ = parser.parse_args()
+
+        metrics = LLMMetrics(
+            request_throughputs=[123],
+            request_latencies=[4, 5, 6],
+            time_to_first_tokens=[7, 8, 9],
+            inter_token_latencies=[10, 11, 12],
+            output_token_throughputs=[456],
+            output_sequence_lengths=[1, 2, 3],
+            input_sequence_lengths=[5, 6, 7],
+        )
+
+        telemetry_metrics = TelemetryMetrics(
+            gpu_power_usage={"gpu0": [45.2, 46.5]},
+            gpu_power_limit={"gpu0": [250.0, 250.0]},
+            energy_consumption={"gpu0": [0.5, 0.6]},
+            gpu_utilization={"gpu0": [75.0, 80.0]},
+            total_gpu_memory={"gpu0": [8.0, 8.0]},
+            gpu_memory_used={"gpu0": [4.0, 4.0]},
+        )
+
+        stats = Statistics(metrics=metrics)
+        telemetry_stats = TelemetryStatistics(telemetry_metrics)
+
+        config = ExporterConfig()
+        config.stats = stats.stats_dict
+        config.telemetry_stats = telemetry_stats.stats_dict
+        config.metrics = stats.metrics
+        config.artifact_dir = Path(".")
+        config.args = args
+
+        exporter = CsvExporter(config)
+        exporter.export()
+
+        expected_content = [
```

Comment:
Since other tests already test the non-Telemetry parts, could you update the tests to only test that the telemetry part is included in the final output? That should be the scope of the Telemetry-specific tests, rather than repeating what other tests are checking.